src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs

```diff
@@ -58,29 +63,38 @@ public async Task ManageBizContactsAsync_WithNoExistingContacts_Correctly()
 
         BizContactUpsertCmd cmd = new((Guid)biz.accountid, (Guid)app.spd_applicationid, new List<BizContactResp>());
 
-        // Action
-        var response = await _bizContactRepo.ManageBizContactsAsync(cmd, CancellationToken.None);
-
-        // Assert
-        var contact = await _context.spd_businesscontacts
-            .Where(c => c._spd_organizationid_value == biz.accountid && c._spd_application_value == app.spd_applicationid)
-            .FirstOrDefaultAsync(CancellationToken.None);
-        Assert.Equal(null, contact);
-
-        //Annihilate
-        _context.DeleteObject(app);
-        _context.DeleteObject(biz);
-        await _context.SaveChangesAsync(CancellationToken.None);
+        try
+        {
+            // Action
+            var response = await _bizContactRepo.ManageBizContactsAsync(cmd, CancellationToken.None);
+
+            // Assert
+            var contact = await _context.spd_businesscontacts
+                .Where(c => c._spd_organizationid_value == biz.accountid)
+                .FirstOrDefaultAsync(CancellationToken.None);
+            Assert.Equal(null, contact);
+        }
+        finally
+        {
+            //Annihilate
+            _context.DeleteObject(app);
+            _context.DeleteObject(biz);
+            await _context.SaveChangesAsync(CancellationToken.None);
+        }
     }
 
     [Fact]
     public async Task ManageBizContactsAsync_WithExistingContacts_Correctly()
     {
+        /********************************************* Dynamics change the schema, licence can only be created when it links to a case. But it seems we cannot create case(incident) in program. So, make this a todo.

```

Comment:
Do we have a related ticket for this "to do"?

Reply:
https://justice.gov.bc.ca/jira/browse/SPDBT-2716 is created for this.